src/SprykerEco/Zed/Heidelpay/Persistence/Propel/Schema/spy_heidelpay.schema.xml

```diff
@@ -93,4 +93,37 @@
         </index>
     </table>
 
+    <table name="spy_payment_heidelpay_notification">
```

Comment:
as long as it a new table, please name it according to our conventions

Reply:
I don't think that is a good ideal. In this scenario we won't have all heidelpay related tables in the same place.